oak-blob-cloud-azure/pom.xml

```diff
@@ -68,6 +71,14 @@
                             azure-core,
                             azure-identity,
                             azure-json,
+                            azure-xml,
+                            azure-storage-blob,
+                            azure-storage-common,
+                            azure-storage-internal-avro,
+                            com.microsoft.aad,
+                            com.microsoft.aad.msal4jextensions,
+                            com.microsoft.aad.msal4jextensions.persistence,
+                            jackson-dataformat-xml,
```

Comment:
Why do we need to embed all ot this? For instance, jackson???

Reply:
Well, that information is insuffiicient.

It just mint hint that deploying this in an OSGi environment requires additional components. Find out which they are and deploy them.

Optimally, include the component in OsgiIT and see what happens (that's the reason why we have these tests).